test/test-process-title.c

```diff
@@ -74,3 +74,65 @@ TEST_IMPL(process_title) {
 
   return 0;
 }
+
+
+static void exit_cb(uv_process_t* process, int64_t status, int signo) {
+  ASSERT(status == 0);
+  ASSERT(signo == 0);
+  uv_close((uv_handle_t*) process, NULL);
+}
+
+
+TEST_IMPL(process_title_big_argv) {
+  uv_process_options_t options;
+  uv_process_t process;
+  size_t exepath_size;
+  char exepath[1024];
+  char jumbo[512];
+  char* args[5];
+
+  /* Remove once https://github.com/libuv/libuv/issues/2667 is fixed. */
+  uv_set_process_title("run-tests");
+
+  exepath_size = sizeof(exepath) - 1;
+#ifdef _AIX
+  if (uv_exepath(exepath, &exepath_size))
+    RETURN_SKIP("uv_exepath() fails spuriously on AIX");
+#else
```

Comment:
`uv_exepath()` is failing on AIX due to the `uv_set_process_title("run-tests")` line above (i.e. it succeeds if the `uv_set_process_title()` call is commented out). It's quite possibly a real latent defect that we haven't spotted before as none of the existing tests that use `uv_exepath()` are using `uv_set_process_title()` but it's kind of late here and I haven't had time to look into it further.

Reply:
I opened #2674 to investigate and tweaked the code to not call `uv_set_process_title()` on AIX.